cvat/apps/engine/migrations/0038_manifest.py

```diff
@@ -1,48 +1,120 @@
 # Generated by Django 3.1.1 on 2021-02-20 08:36
 
 import glob
+import itertools
+import logging
 import os
+import sys
 from re import search
 
 from django.conf import settings
 from django.db import migrations
 
 from cvat.apps.engine.models import (DimensionType, StorageChoice,
                                      StorageMethodChoice)
+from cvat.apps.engine.media_extractors import get_mime
 from utils.dataset_manifest import ImageManifestManager, VideoManifestManager
 
-def migrate_data(apps, shema_editor):
+def get_logger():
+    migration = os.path.basename(__file__).split(".")[0]
+    logger = logging.getLogger(name=migration)
+    logger.setLevel(logging.INFO)
```

Comment:
We have a special file with log configs. Do you think we can put the code into https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/engine/log.py?